rpc/core/routes.go

```diff
@@ -58,4 +58,5 @@ func (env *Environment) AddUnsafe(routes RoutesMap) {
 	routes["dial_seeds"] = rpc.NewRPCFunc(env.UnsafeDialSeeds, "seeds", false)
 	routes["dial_peers"] = rpc.NewRPCFunc(env.UnsafeDialPeers, "peers,persistent,unconditional,private", false)
 	routes["unsafe_flush_mempool"] = rpc.NewRPCFunc(env.UnsafeFlushMempool, "", false)
+	routes["resync"] = rpc.NewRPCFunc(env.UnsafeResync, "start_height,end_height", false)
```

Comment:
reindex or something more related to events may be a better name. This could be confused with resyncing blocks from the network.

Reply:
Sure, I'll change to `reindex` and the relate docs.